CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to Stencil
```

Comment:
How about we open up the issue queue so people can report bugs or search on issues & they resolution or any PR work being done on it.